src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,77 @@
  */
 
 public class Plane {
+    private String model;
+    private String color;
+    private int weight;
+    private int maxSpeed;
+    private int passengerCapacity;
 
+    public Plane(PlaneBuilder planeBuilder) {
+        this.model = planeBuilder.model;
+        this.color = planeBuilder.color;
+        this.weight = planeBuilder.weight;
+        this.maxSpeed = planeBuilder.maxSpeed;
+        this.passengerCapacity = planeBuilder.passengerCapacity;
+    }
+
+    public String getModel() {
+        return this.model;
+    }
+
+    public String getColor() {
+        return this.color;
+    }
+
+    public int getWeight() {
+        return this.weight;
+    }
+
+    public int getMaxSpeed() {
+        return this.maxSpeed;
+    }
+
+    public int getPassengerCapacity() {
+        return this.passengerCapacity;
+    }
+
+    public static class PlaneBuilder {
+        private String model;
+        private String color;
+        private int weight;
+        private int maxSpeed;
+        private int passengerCapacity;
+
+        public PlaneBuilder() {
+        }
```

Comment:
do we need to create this type of constructor or we have it by default?

Reply:
i suggest well get it by default too :) ty